.github/workflows/CI_master.yml

```diff
@@ -0,0 +1,67 @@
+# ***************************************************************************
+# *   Copyright (c) 2023 0penBrain                               *
```

Comment:
fix whitespace

Reply:
It was copied from other files. All headers are like this in FC.
If this needs a fix, let's do it in a batch.

Reply:
Good to know. Copied from where?

Reply:
Another file of the codebase. :) Can't remember which one I had opened when I did that. ;)